examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,240 @@
+                            --assume-role-policy-document '{
+            "Version": "2012-10-17",
+            "Statement": [
+            {
+                "Sid": "",
+                "Effect": "Allow",
+                    "Principal":
+                    {
+                    "Service": "monitoring.rds.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+
+            }
+        ]
+        }'
+ ```
+ (Attach the IAM role with AmazonRDSEnhancedMonitoringRole policy.)
+
+```shell
+    
+    aws iam attach-role-policy --role-name aurora-monitoring \
+                           --policy-arn arn:aws:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole
+
+```
+
+  Create an 2nd IAM role and attach policy to the role (This role is required for aurora proxy to fetch db secrets from secret manager )
+
+```shell
+    aws iam create-role --role-name rds-proxy \
+                        --assume-role-policy-document '{    
+        "Version": "2012-10-17",
+        "Statement": [
+        {
+            "Sid": "",
+            "Effect": "Allow",
+                "Principal":
+                {
+                "Service": "rds.amazonaws.com"
+                },
+                "Action": "sts:AssumeRole"
+
+        }
+    ]
+    }'
+```
+
+ ```shell
+
+    aws iam put-role-policy --role-name rds-proxy \
+                            --policy-name rds-proxy-policy \  
+                            --policy-document '{
+                            "Version": "2012-10-17",
+                            "Statement": [
+                                {
+                                    "Sid": "getsm",
+                                    "Effect": "Allow",
+                                    "Action": "secretsmanager:GetSecretValue",
+                                    "Resource": "*"
+                                },
+                                {
+                                    "Sid": "kmsdecrypt",
+                                    "Effect": "Allow",
+                                    "Action": "kms:Decrypt",
+                                    "Resource": "*",
+                                    "Condition": {
+                                        "StringEquals": {
+                                            "kms:ViaService": "secretsmanager.us-east-1.amazonaws.com"
+                                        }
+                                    }
+                                }
+                            ]
+                        }'
+  
+```
+  ### Provide value to the claim before applying.
+     
+     1. Open the claim and substitute aurora-monitoring role arn with monitoringRoleArn and rds-proxy role arn with proxyRoleArn respectively.
+     2. Make sure check the CIDR block for application and provide in the claim. This will used as the security group 
+        ingress rule for proxy from application ( we can also provide the security group id instead of CIDR).    
```

Comment:
Should be placed outside code block. It's harder to read

Reply:
updated .